Tests/UrlFormEncodingTests/UrlFormDecoderTests.swift

```diff
@@ -0,0 +1,133 @@
+import Prelude
+import SnapshotTesting
+import UrlFormEncoding
+import XCTest
+
+final class UrlFormDecoderTests: XCTestCase {
```

Comment:
These tests provide a bunch of examples! Would be fun to get some kind of property testing here, though.

Reply:
we gotta add these to the `imports` var in the `Makefile` for linux :/ just did it here 96dab4b

Reply:
oops never mind, I thought this was in a new module! reverted...